open-ce/docker_build.py

```diff
@@ -112,26 +113,22 @@ def build_in_container(image_name, output_folder, arg_strings):
 
     result = _create_container(container_name, image_name, output_folder)
     if result:
-        print("Error creating docker container: " + container_name)
-        return result
+        raise OpenCEError("Error creating docker container: \"{}\"".format(container_name))
```

Comment:
Do you think the smaller functions such as `_create_container` should be throwing the exception? I think the code would be a little cleaner, but maybe the error messages wouldn't be as specific.

Reply:
I though it made sense for the internal functions to return error codes, since they're basically just simple wrappers for a system call. Let me take another look at them and see how much it cleans it up to switch those as well.

Reply:
This is done, it is a lot cleaner this way.